### PR TITLE
Add example Docker Compose snippet

### PR DESCRIPTION
**Pull request by sethforprivacy**

### CODE REVIEW

File: .github/workflows/examples/docker-compose.yml
Comment (AngusP) [inline .github/workflows/examples/docker-compose.yml:1]: I think you meant to put this in `/examples` not within `.github/`?

**Comment by AngusP** [comment]
Also needs rebase as I've merged #1

File: .github/workflows/examples/docker-compose.yml
Comment (sethforprivacy) [reply .github/workflows/examples/docker-compose.yml:1]: Oops, absolutely.

**Comment by sethforprivacy** [comment]
> Also needs rebase as I've merged #1

Fixed and rebased.

**Review by AngusP — APPROVED** [review]

**Pull request merged**